rockyjs/app/simply-rocky.js

```diff
@@ -0,0 +1,1462 @@
+var Color = require('../../src/js/lib/color');
```

Comment:
This is a copy and paste from simply-rocky.js

It is intended to be culled down as more components are implemented